src/olympia/addons/tests/test_views.py

```diff
@@ -1704,6 +1704,8 @@ def test_get_addon_not_found(self):
 
 
 class TestAddonViewSetDetail(AddonAndVersionViewSetDetailMixin, TestCase):
+    client_class = APITestClient
```

Comment:
fair enough, let's keep it this way until it hurts, I don't have any better ideas right now too